cmd/kpod/docker/types.go

```diff
@@ -0,0 +1,271 @@
+package docker
```

Comment:
Just a generic question to Nalin/Dan, we seem to have a fair amount of common code between skopeo, buildah and now kpod.  I understand not pulling from upstream, but might it make sense to create a new shared project to pull from that we'd control?